helpers/answerExercises.js

```diff
@@ -57,30 +60,7 @@ async function markAsCompleted(user, exerciseIdFromFunction, pointsFromFunction,
       {
         arrayFilters: [{ 'section.sectionId': sectionId }]
       }
-    ).post(async function() {
```

Comment:
I approve, but make sure that you are suppose to delete this and that it is not a mistake ... 👍

Reply:
This change is from the most recent pull request to backend dev from gamified, so im pretty sure it is ok